Samples/V1.0Samples/LocalMediaSamples/PolicyRecordingBot/README.md

```diff
@@ -57,25 +59,138 @@ To verify your policy was assigned correctly:
+   * Domain Name: The domain name (domainNameLabel) is optional during initial creation and can be specified or updated later.
+   
+   Match Public IP Name:
+   * Ensure that the name attribute under <PublicIP> (MyPublicIP in this example) matches the name used in your application code to fetch the public IP address dynamically.
+   * To ensure that the domainNameLabel matches between your configuration (.cscfg file) and the Azure portal settings.
+   
+Step 4: Create Your Cloud Service (Extended Support)
+      You can deploy your Azure extended service using various methods, such as the Azure portal, PowerShell, Azure Resource Manager templates, or Visual Studio.
+
+1. To create a Cloud Service (Extended Support) in Azure, follow these steps:
+   
+### Note: 
+
+   1. Configure Storage Account for Configuration Files.
+    * To store configuration files for your Azure extended service, you'll need to set up a storage account. Follow these steps to configure the storage account:(https://learn.microsoft.com/en-us/azure/storage/common/storage-account-create?tabs=azure-portal).
+
+   2. Package Your Cloud Service for Deployment
+    * Before you can create your Azure extended service, you need to package your cloud service application to include configuration files and dependencies.
+    * Right click PolicyRecordingBot, then click `Package...`.
+
+   Option 1: Upload to Azure Storage Account:
+    * Upload your packaged application (cspkg file) along with the .cscfg and .csdef files to an Azure Storage Account container.
+    * This method allows you to deploy directly from the Azure Storage Account during service creation.
+
+   Option 2: Use Local Files:
+    * Alternatively, you can deploy directly from local files during the service creation process.
+    * Ensure all required files, including .cscfg and .csdef, are accessible and correctly referenced during deployment.
+
+  Create Cloud Service (extended support):
+   * Use the Azure portal to create a Cloud Service (Extended Support). Follow this guide: Create a Cloud Service (Extended Support).(https://learn.microsoft.com/en-us/azure/cloud-services-extended-support/deploy-portal)
 
-2. Set up SSL certificate and upload to the cloud service
-    1. Create a wildcard certificate for your service. This certificate should not be a self-signed certificate. For instance, if your bot is hosted at `bot.contoso.com`, create the certificate for `*.contoso.com`.
-    2. Upload the certificate to the cloud service.
-    3. Copy the thumbprint for later.
+  Obtain Your Public IP DNS name:
+   After the service is created, obtain the "Public IP DNS name" from the Azure portal. This URL will serve as your DNS name and Common Name (CN) for further configurations (e.g. bot.contoso.com).
+   ![Public IP DNS name](Images/PublicIPDNSName.png).
 
-3. Set up cloud service configuration
+2. Publish a Cloud Services (classic) project to Cloud Services (extended support) by using Visual Studio:
```

Comment:
I am confused on the numbering here. The last numbering used before this is also 2